src/rest/Facet/FacetResponse.ts

```diff
@@ -29,6 +29,11 @@ export interface IFacetResponseValue extends IRangeValue {
    * The children of this hierarchical facet value.
    */
   children?: IFacetResponseValue[];
+
+  /**
+   * Whether the hierarchical value is a childless node.
+   */
+  isLeafValue?: boolean;
```

Comment:
Not sure I understand this property. From what I understand, this is true when this facet has children and false otherwise.

Reply:
Exactly

Reply:
I think a better description would explain when the property is `true`. e.g.

```
When the hierarchical value has no children, this property is `true`.
```

Reply:
Is this property only used for simplicity? or is there an other reason to not use children.length to check if it's a leaf or not?

Reply:
This property, and its description, come from the API/index. They are telling us in advance that if you try to get this values children, you will see results, this way we can display the > arrow.